cirq-core/cirq/ops/linear_combinations.py

```diff
@@ -498,6 +498,23 @@ def _unitary_(self) -> np.ndarray:
             return m
         raise ValueError(f'{self} is not unitary')
 
+    def _json_dict_(self):
+        def key_json(k):
+            # k is a frozenset, each element of frozen set is a tuple.
```

Comment:
use type annotations instead of a comment?

Reply:
done,